app/assets/javascripts/admin/orders/controllers/orders_controller.js.coffee

```diff
@@ -52,3 +52,11 @@ angular.module("admin.orders").controller "ordersCtrl", ($scope, RequestMonitor,
   $scope.changePage = (newPage) ->
     $scope.page = newPage
     $scope.fetchResults(newPage)
+
+  $scope.bulkInvoice = ->
+    params = ''
+    angular.forEach $scope.selected, (selected, order_id) ->
+      params += 'order_ids[]='+order_id+'&' if selected
+
+    $window.open('/admin/orders/bulk_invoice?'+params)
+    true
```

Comment:
This `true` would normally be unnecessary, but I encountered a weird Coffeescript bug that was throwing JS console errors, and this was the fix. 

Outlined here: https://github.com/angular/angular.js/issues/4853#issuecomment-28491586